xcdat/regridder/regrid2.py

```diff
@@ -30,7 +34,15 @@ def __init__(
             Dataset containing the source grid.
         output_grid : xr.Dataset
             Dataset containing the destination grid.
-        options : Any
+        unmapped_to_nan : bool
+            If True, unmapped values are set to NaN. Default is True.
+        output_weights : bool | str
+            If True, output weights are added to the output dataset as weights.
+            If str, the name of the variable to store the weights. Default is False.
+        create_nan_mask : bool
+            If True, a mask is created using the nan values from source variable. If
+            a mask already exists in the Dataset it will be ignored.
```

Comment:
```suggestion
            If True, a mask is created using the nan values from source variable. If
            a mask already exists in the Dataset it will be ignored. Default is False.
```